frontend/src/components/tokenHandler.js

```diff
@@ -0,0 +1,32 @@
+import { useContext} from "react";
+import  {TokenContext}  from "../Authenticator";
+
+
+
+
+export const useTokenStateHandler = () => {
+    const [ token, setToken ] = useContext(TokenContext);
```

Comment:
Esetleg első futásnál kiolvashatná a sessionStorage-ból és beállíthatná a contextben, use case: frissen nyitott lap.

Reply:
Ez csak a localstorage-al működik. Legyen az?